test/functional/fixtures/regression/gh-2325/test.js

```diff
@@ -0,0 +1,5 @@
+describe('[Regression](GH-2325)', function () {
```

Comment:
You can remove the functional test since you've refactored it into the client test.